multi-agent/tests/e2e/test_session_stress_submit.py

```
@@ -0,0 +1,1442 @@
+        """
+        Concurrent create + submit/poll until COMPLETED.
+        """
+        blueprint_id = None
+        owns_blueprint = False
+        session_ids = []
+        
+        try:
+            print(f"\n{'=' * 80}")
+            print("🧪 STARTING E2E SESSION STRESS TEST (submit + poll)")
+            print(f"{'=' * 80}")
+            print(f"Configuration:")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)